includes/backends/class-omise-backend-installment.php

```diff
@@ -53,6 +53,13 @@ public function initiate() {
 				'interest_rate'      => 0.65,
 				'min_allowed_amount' => 500.00,
 			),
+
+			'installment_scb' => array(
```

Comment:
Maybe it would be a good idea to add a test case for this in `tests/unit/includes/backends/class-omise-backend-installment-test.php` ?

Reply:
@alan-hadyk Added in https://github.com/omise/omise-woocommerce/pull/193/commits/9ece19785b0241f17a870d19fd3f65cb24f47c4d.